GitUI/CommandsDialogs/FormRebase.Designer.cs

```diff
@@ -542,30 +571,34 @@ private void InitializeComponent()
             this.rebasePanel.Controls.Add(this.Branches);
             this.rebasePanel.Controls.Add(this.ShowOptions);
             this.rebasePanel.Dock = System.Windows.Forms.DockStyle.Fill;
-            this.rebasePanel.Location = new System.Drawing.Point(3, 51);
+            this.rebasePanel.Location = new System.Drawing.Point(3, 39);
             this.rebasePanel.Name = "rebasePanel";
-            this.rebasePanel.Size = new System.Drawing.Size(503, 27);
+            this.rebasePanel.Size = new System.Drawing.Size(678, 27);
             this.rebasePanel.TabIndex = 32;
             // 
-            // Commit
+            // chkCommitterDateIsAuthorDate
             // 
-            this.Commit.Location = new System.Drawing.Point(3, 198);
-            this.Commit.Name = "Commit";
-            this.Commit.Size = new System.Drawing.Size(162, 25);
-            this.Commit.TabIndex = 23;
-            this.Commit.Text = "Commit...";
-            this.Commit.UseVisualStyleBackColor = true;
-            this.Commit.Click += new System.EventHandler(this.CommitButtonClick);
+            this.chkCommitterDateIsAuthorDate.Anchor = System.Windows.Forms.AnchorStyles.Left;
+            this.chkCommitterDateIsAuthorDate.AutoSize = true;
+            this.chkCommitterDateIsAuthorDate.Location = new System.Drawing.Point(495, 3);
+            this.chkCommitterDateIsAuthorDate.Name = "chkCommitterDateIsAuthorDate";
+            this.chkCommitterDateIsAuthorDate.Size = new System.Drawing.Size(169, 17);
+            this.chkCommitterDateIsAuthorDate.TabIndex = 10;
+            this.chkCommitterDateIsAuthorDate.Text = "Committer date is author date";
+            this.toolTip1.SetToolTip(this.chkCommitterDateIsAuthorDate, "Changes the committer dates of the commits in this branch\r\ninto the author dates." +
+        "\r\n");
+            this.chkCommitterDateIsAuthorDate.UseVisualStyleBackColor = true;
+            this.chkCommitterDateIsAuthorDate.Click += new System.EventHandler(this.DateClick);
```

Comment:
Please don't reuse the same handler, create a new one and call a shared method. Otherwise it makes it extremely difficult to reason about the code.
```suggestion
            this.chkCommitterDateIsAuthorDate.Click += new System.EventHandler(this.chkCommitterDateIsAuthorDate_Click);
```